modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/validation/util/DateParameterUtil.java

```diff
@@ -62,8 +68,25 @@ public static LocalDateTime getLocalDateTime(String dateTimeString) {
 			return null;
 		}
 
-		return LocalDateTime.parse(
-			dateTimeString, DateTimeFormatter.ofPattern("yyyy-MM-dd H:mm"));
+		String parseException = null;
+
+		for (String datePattern : datePatterns) {
+			try {
+				return LocalDateTime.parse(
+					dateTimeString, DateTimeFormatter.ofPattern(datePattern));
+			}
+			catch (DateTimeParseException dateTimeParseException) {
+				parseException = String.valueOf(dateTimeParseException);
+			}
+		}
+
+		if (parseException != null) {
+			_log.error(parseException);
+		}
```

Comment:
Since we are now checking for multiple date patterns, this log error would only save the last datePattern tried (it would iterate between all patterns in the list and then, if all of them failed, would print this error), but this is misleading, since the person would only see that last error, and not that it failed on all parse tries. Maybe we should create a more meaningful error message, showing that the dateTimeString given is not valid in any of the patterns.